src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -175,41 +64,21 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
                                                    const SelectionChangedEventArgs& args)
```

Comment:
QQ: not sure why this logic is still in the Page -- the selected color scheme is a property of the ViewModel, and changing it should cascade through the viewmodel?

Reply:
Removed!